FrontEnd/global.scss

```diff
@@ -0,0 +1,11 @@
+:root {
+    --gold-primary: #BB8F52;
+    --gold-accent: rgba(201, 170, 136, 30);
+
+    --white: #ffffff;
+    --black: #333333;
+}
```

Comment:
if there's going to be much more of these, can we have this in a separate "colors.scss" file please?
(which we can pull / import to "global.scss")